packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss

```diff
@@ -1,13 +1,17 @@
 @import '../../common/common';
 
 $iconForegroundColor: $ms-color-neutralSecondary;
-$actionableHeaderForegroundColor: $ms-color-neutralSecondary;
+$headerForegroundColor: $ms-color-neutralPrimary;
+$headerBackgroundColor: $ms-color-white;
```

Comment:
Spoke with our designer, we want to have a background color of white for headers.